production/helm/loki/src/helm-test/Dockerfile

```diff
@@ -7,7 +7,7 @@ COPY . /src/loki
 WORKDIR /src/loki
 RUN make clean && make BUILD_IN_CONTAINER=false helm-test
 
-FROM alpine:3.16.5
-RUN apk add --update --no-cache ca-certificates=20220614-r0
+FROM alpine:3.16.7
+RUN apk add --update --no-cache ca-certificates=20230506-r0
```

Comment:
I tried removing the specific version of this package, our helm lint doesn't let me do that for some reason!